Python/Product/BuildTasks/Microsoft.PythonTools.targets

```diff
@@ -168,7 +168,7 @@ permissions and limitations under the License.
     <CreatePythonCommandItem Target="pylint"
                              TargetType="module"
                              Arguments="&quot;--msg-template={abspath}({line},{column}): {category} {msg_id}: {msg} [{C}:{symbol}]&quot; -r n @(Compile, ' ')"
-                             WorkingDirectory="$(MSBuildProjectDirectory)"
+                             WorkingDirectory="$(QualifiedProjectHome)"
```

Comment:
what is QualifedProjecthome actually represent?

Reply:
defined above. looks like it adds a ending backslash or the projectHome dir

Reply:
It's the `ProjectHome` of the pyproj. When you open from existing code, `ProjectHome` gets set to something other than `.`. But `MSBuildProjectDirectory` always points to the directory containing your pyproj, even if the actual code files live somewhere else.

So that's why this is currently broken when creating a project from existing code, since the `MSBuildProjectDirectory` will have no python files underneath, which causes pylint and mypy to blow up when trying to run them.

Reply:
Here's where it gets set, on line 135:

```
    <QualifiedProjectHome Condition="'$(ProjectHome)' != '' and '$(ProjectHome)' != '.'">$([System.IO.Path]::GetFullPath($([System.IO.Path]::Combine($(MSBuildProjectDirectory), $(ProjectHome)))))</QualifiedProjectHome>
```